sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessageSerializer.java

```diff
@@ -225,6 +234,62 @@ public int getSize(Message amqpMessage) {
         return amqpMessage;
     }
 
+    private Map<String, Object> convertToDescribedType(Map<String, Object> propertiesValue) {
```

Comment:
I believe this can be static? It doesn't look like it depends on any instance variables.